tests/models/whisper/test_tokenization_whisper.py

```diff
@@ -499,25 +499,3 @@ def test_offset_decoding(self):
 
         output = multilingual_tokenizer.decode(INPUT_TOKENS, output_offsets=True)["offsets"]
         self.assertEqual(output, [])
-
-    @require_jinja
-    def test_tokenization_for_chat(self):
```

Comment:
Chat template doesn't make sense for Whisper (a speech recognition model) - have removed the test to keep the CI lightweight (cc @Rocketknight1)

Reply:
Fine with me!